Docs/landing/data/releases.toml

```diff
@@ -1,7 +1,7 @@
 current = "2.9.3"
 
 [[versions]]
-  version = "2.10.0-beta1"
+  version = "2.10.0"
```

Comment:
Let's revert this until ready to do the release

Reply:
done